app/models/updateVatSubscription/request/RequestedChanges.scala

```diff
@@ -42,7 +41,7 @@ object ChangeCommsPreferenceAndEmail extends RequestedChanges(commsPreference =
 
 object RequestedChanges {
 
-  val DESApi1365Writes: AppConfig => Writes[RequestedChanges] = appConfig => Writes { model =>
+  val DESApi1365Writes: AppConfig => Writes[RequestedChanges] = _ => Writes { model =>
```

Comment:
Underscoring the `AppConfig` param here means it's still there but just not being given a name. Looks like we have no need for `AppConfig` in this reads anymore so can we remove it instead? i.e. converting this function from:
`AppConfig => Writes[RequestedChanges] = _ => Writes`...
to
`Writes[RequestedChanges] = Writes`...